docs/partner_editable/test_deployment.adoc

```diff
@@ -0,0 +1,2 @@
+// Add steps as necessary for accessing the software, post-configuration, and testing. Don’t include full usage instructions for your software, but add links to your product documentation for that information.
+<Add instructions for testing the deployment.>
```

Comment:
Are we shipping with this in?

Reply:
Right now it isn't pulled in the docs, but I can remove it